Social share icon styling

urchykli opened the pull request:
Closes #50 

Styled social share icons based on mock. Updated size, color, hover transition and spacing. 

I also added a class name to the print button, as it needed additional styling to remove default

jnschrag commented on src/shared/site-config/Header.js:
```diff
@@ -12,6 +12,7 @@ const Header = () => {
             alt="Center for Strategic and International Studies"
             title="Center for Strategic and International Studies"
           />
+          <span className="site-header__logo-divider">|</span> Explore The Data
```
Instead of including the vertical rule as an actual element in the HTML when it's only necessary for visual purposes, you can do one of two things:

1. You can set a border on the SVG & use padding & margin to position it
2. You can make this `<span className="site-header__tagline">Explore the Data</span>` and use a pseudo element to place the bar (a border won't work here because they're not the same height).

urchykli commented on src/scss/components/_social-share.scss:
```diff
@@ -0,0 +1,38 @@
+  &-item {
+    display: flex;
+    justify-content: center;
+    width: 37px;
```
I updated the icons, but replacing width and height on the li (-item) with font size didn't work.

urchykli commented on src/scss/components/_social-share.scss:
```diff
@@ -0,0 +1,38 @@
+  }
+
+  .icon {
+    width: 20px;
```
Oh, yes, this is what I did. Thanks!

urchykli commented on src/scss/components/_social-share.scss:
```diff
@@ -0,0 +1,38 @@
+  list-style-type: none;
+
+  &-item {
+    display: flex;
```
I tried to remove this, but it throws off the alignment of the print icon.

jnschrag replied:
If you remove the display flex, the width, and the height properties on the .share-item class, everything lines up as intended.

urchykli commented:
I made the requested changes, but please let me know if you see anything else that needs adjusting.  I will style the rest of the header via issue #43. Thanks!

jnschrag commented on src/scss/layout/_header.scss:
```diff
@@ -1 +1,27 @@
+
+  // logo link
+  &__logo {
+    display: block;
```
This should be `display: flex;` with an `align-items: center` to line up the SVG & text. Although if you want to do that as a part of the larger header styles, I'm good with that!

urchykli replied:
Thanks for the guidance on these!

jnschrag commented on src/shared/site-config/SocialShare.js:
```diff
@@ -7,31 +7,37 @@ const SocialShare = () => {
       <li className="share-item">
         <a
           href={`https://www.facebook.com/sharer.php?u=${window.location.href}`}
+          className="share-link"
```
Technically, for proper BEM naming, these should be: `share__item` and `share__link`.

urchykli commented:
Hi Jacque - I've updated the following. Please let me know if I need to adjust anything else.  Thanks for your help with this!
- Update class names to follow BEM naming conventions
- Remove formatting on share item class
- Added styling to align logo and text
